src/app.tsx

```diff
@@ -1,21 +1,18 @@
 import InstagramPost from './components/instagramPost/InstagramPost';
-
-import {
-    SAMPLE_POST_IMAGE,
-    SAMPLE_PROFILE_PICTURE,
-    SAMPLE_POST_DESCRIPTION,
-    SAMPLE_USER_NAME,
-    SAMPLE_DATE,
-} from './constants';
+import { MOCK_DATA, POST_IMAGE_SAMPLES, SAMPLES_PROFILE_PICTURE } from './mockText';
```

Comment:
SAMPLE_IMAGES

Reply:
SAMPLE_PROFILE_PICTURES

Reply:
even tho the file only has strings i wouldnt all this mock text.

its text that points to data.

call it mock-data